lib/rules/no-useless-escape.js

```diff
@@ -61,20 +61,38 @@ var VALID_REGEX_ESCAPES = [
 module.exports = function(context) {
 
     /**
+    * Returns location of the unnecessary escape
+    *
+    * @param {ASTNode} node - the node containing the unnecessary escape
+    * @param {int} indexInNode - the index of unnecessary escape inside node
+    * @returns {Object} - the location of the unnecessary escape
+    */
+    function getLocation(node, indexInNode) {
```

Comment:
I don't think we need a function for this since it is pretty straightforward and used only in one place.